src/fido2.js

```diff
@@ -0,0 +1,147 @@
+const DJANGO_FIDO_FORM_ID = 'django-fido-form'
```

Comment:
Can we move the JS source to `static` directory? I don't quite like the `src` directory, since it only contains the JS and not the `django_fido` itself.

Reply:
See my discussion above. I proposed to move it to `assets/js` directory, the same location as in Ferda.

Reply:
We agreed on `assets/js` for JavaScript sources, `assets/css` for CSS sources and so on. The transpiled files are placed under the static files directory.

Reply:
This is not relevant, we agreed on move to django_fido/js and js-tests. This move is already done.